Show me image/routine.cpp

```diff
@@ -68,9 +68,9 @@ bool init() {
 	SDL_GL_SetAttribute(SDL_GL_ACCUM_GREEN_SIZE,  8);
 	SDL_GL_SetAttribute(SDL_GL_ACCUM_BLUE_SIZE,   8);
 	SDL_GL_SetAttribute(SDL_GL_ACCUM_ALPHA_SIZE,  8);
- 
-	SDL_GL_SetAttribute(SDL_GL_MULTISAMPLEBUFFERS,  1);
-	SDL_GL_SetAttribute(SDL_GL_MULTISAMPLESAMPLES,  2);
+
+	SDL_GL_SetAttribute(SDL_GL_MULTISAMPLEBUFFERS,  0);
+	SDL_GL_SetAttribute(SDL_GL_MULTISAMPLESAMPLES,  0);
```

Comment:
What is it for? Does it generate error?

Reply:
I'm not sure, but it seems that I wasn't able to create a window with this flags enabled. `SDL_SetVideoMode` was returning `NULL` in my case. 
As I understood it's needed for enabling AA in game, I don't know do we need that.

Reply:
Alright. We'll keep it this way for now.